src/Sql/dbo/Stored Procedures/OrganizationSecretsManager_Update.sql

```diff
@@ -0,0 +1,30 @@
+CREATE PROCEDURE [dbo].[OrganizationSecretsManager_Update]
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Plan [nvarchar](50),
+    @PlanType [tinyint],
+    @UserSeats [int],
+    @ServiceAccountSeats [int],
+    @UseEnvironments [bit],
+    @MaxAutoscaleUserSeats [int],
+    @MaxAutoscaleServiceAccounts [int],
+    @MaxProjects [int]
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    UPDATE
+        [dbo].[OrganizationSecretsManager]
+    SET
+        OrganizationId = @OrganizationId,
```

Comment:
I believe we should use `[]` around updates since that seems to be the convention in other stored procedures